fairseq/models/wav2vec/wav2vec2_scribblelens.py

```diff
@@ -403,6 +417,11 @@ def upgrade_state_dict_named(self, state_dict, name):
         """Upgrade a (possibly old) state dict for new versions of fairseq."""
         return state_dict
 
+    def to(self, *args, **kwargs):
+        self = super().to(*args, **kwargs) 
+        self.smartpooling_filters = self.smartpooling_filters.to(*args, **kwargs) 
```

Comment:
To nie będzie potrzebne, jesli smartpooling filters beda zadeklarowane jako buffer, (tensor, ale nieuczony) albo parameter (tensor, uczony przez backprop)